interface.py

```diff
@@ -126,7 +127,7 @@ def get_dealt_cards(card_pack):
     """
     # Maximum number of cards is 9.
     # Robot's damages reduce the count of dealt cards - each damage one card.
-    dealt_cards_count = MAX_CARD_COUNT-robot_data.damages
+    dealt_cards_count = MAX_CARD_COUNT-robot.damages
```

Comment:
Odkud se bere tohle `robot`?

Reply:
to je robot který je v interfacestate, tedy se přiřadí když ho server pošle klientovi. nebo ne?

Reply:
Ne. Tohle je globální proměnná `robot`, ne součást stavu (`interface_state.robot` nebo `self.robot`). Tahle funkce ani žádný interfacestate nemá k dispozici.

Reply:
A kdyz tam byly robot_data tak to bylo dobře?  Ja to jen přejmenovala.navic se to přesouvá do backendu, takze uz to tu nebude, viz karolinin PR.  Ale uz vim, tady byl v tom souboru jeste nahore robot = ... Nějaký fiktivní konkretni takze jo, to se ještě musi zmenit.

Reply:
Nebylo, chyběl tam pořád argument. Ale pro ten PR bych to nechala, zpracuji přenesení a vyřeším ten problém.